### ISSUE TITLE
First downloadTCGA function examples

### ISSUE THREAD

**Issue by MarcinKosinski** [MarcinKosinski/RTCGA#5]
Try this. Works fine for me.

````{Ruby}
> library(RTCGA)
> downloadTCGA( cancerTypes = "BRCA", additionalInfo = "miR_gene_expression", 
+               destDir = "hre/" )
trying URL 'http://gdac.broadinstitute.org/runs/stddata__2015_02_04/data/BRCA/20150204/gdac.broadinstitute.org_BRCA-FFPE.Merge_mirnaseq__illuminahiseq_mirnaseq__bcgsc_ca__Level_3__miR_gene_expression__data.Level_3.2015020400.0.0.tar.gz'
Content type 'application/x-gzip' length 51580 bytes (50 Kb)
opened URL
downloaded 50 Kb

BRCA 
   0 
> downloadTCGA( cancerTypes = c("BRCA", "OV"), destDir = "hre/" )
trying URL 'http://gdac.broadinstitute.org/runs/stddata__2015_02_04/data/BRCA/20150204/gdac.broadinstitute.org_BRCA.Merge_Clinical.Level_1.2015020400.0.0.tar.gz'
Content type 'application/x-gzip' length 2189748 bytes (2.1 Mb)
opened URL
downloaded 2.1 Mb

trying URL 'http://gdac.broadinstitute.org/runs/stddata__2015_02_04/data/OV/20150204/gdac.broadinstitute.org_OV.Merge_Clinical.Level_1.2015020400.0.0.tar.gz'
Content type 'application/x-gzip' length 1574476 bytes (1.5 Mb)
opened URL
downloaded 1.5 Mb

BRCA   OV 
   0    0 

````

**Comment by MarcinKosinski** [comment]
Don't know how to eliminate the resulted vector of zeros :/

**Comment by pbiecek** [comment]
fixed in fd111664d545199638213d9a73d3a314b0045d1d
